app/marketing/mails.py

```diff
@@ -1140,10 +1140,8 @@ def reject_faucet_request(fr):
 
 
 def new_bounty_daily(bounties, old_bounties, to_emails=None):
-    from marketing.views import trending_quests
-
-    if not bounties:
-        return
+    # if not bounties:
```

Comment:
Is this commented out code still needed? If not please remove it

Reply:
Done in [3e561b6](https://github.com/gitcoinco/web/pull/6471/commits/3e561b647f26c969686f74c28a11de7b971b2d91)